content/profiles/busayo-amowe.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Olubusayo Amowe"
+profileimage: "/images/profile-pictures/busayo-amowe.jpg"
+coverimage: "/images/profile-pictures/busayo-amowe-cover.jpeg"
+tags: ["CSharp", ".NET",".NET core","Azure","Devops","AKS"]
+twitter: "amoweo"
+github: "amoweolubusayo"
+linkedin : "https://www.linkedin.com/in/amoweolubusayo"
+comments: false
+summary: "I am a .NET Developer and a huge man of MS products."
```

Comment:
IS this a typo? Should it be huge "fan"?

Reply:
My bad lol. I must have been thinking of a man. Will correct it now. Thanks for noticing this

Reply:
@Layla-P fixed the typo